generatelog

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-command=$1
+
```

Comment:
generatelog - Script name is too generic, please change to more appropriate name to reflect exact purpose.